### PR TITLE
refactor(openapi): use python f-string to replace "urljoin"

### PR DESCRIPTION
**Pull request by linjiX**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2dfq55u 对代码仓库使用的 url 进行标准化操作](https://app.clickup.com/t/2dfq55u)

File: graviti/openapi/commit.py [hunk 1]
```diff
@@ -114,7 +113,7 @@ def list_commits(
         }
 
     """
-    url = urljoin(url, f"v2/datasets/{owner}/{dataset}/commits")
+    url = f"{url}v2/datasets/{owner}/{dataset}/commits"
```
Comment (graczhual) [inline graviti/openapi/commit.py:116]: `{url}/v2`

File: graviti/openapi/user.py [hunk 2]
```diff
@@ -34,5 +33,5 @@ def get_current_user(access_key: str, url: str) -> Dict[str, Any]:
         }
 
     """
```
Comment (graczhual) [inline graviti/openapi/user.py:35]: `gas.graviti.com/` -> `api.graviti.com`

**Review by graczhual — CHANGES_REQUESTED** [review]

File: graviti/openapi/dataset.py [hunk 3]
```diff
@@ -55,7 +54,7 @@ def create_dataset(
         }
 
     """
-    url = urljoin(url, "v2/datasets")
+    url = "{url}/v2/datasets"
```
Comment (graczhual) [inline graviti/openapi/dataset.py:57]: `f`

File: graviti/openapi/dataset.py [hunk 4]
```diff
@@ -148,7 +147,7 @@ def list_datasets(
...
```
Comment (graczhual) [inline graviti/openapi/dataset.py:150]: `f`

File: graviti/openapi/user.py [hunk 5]
```diff
@@ -34,5 +33,5 @@ def get_current_user(access_key: str, url: str) -> Dict[str, Any]:
...
-    url = urljoin(url, "v2/current-user")
+    url = "{url}/v2/current-user"
```
Comment (graczhual) [inline graviti/openapi/user.py:36]: `f`

**Review by graczhual — CHANGES_REQUESTED** [review]

**Review by graczhual — APPROVED** [review]

**Pull request merged**